srcpkgs/ffms2/template

```diff
@@ -1,19 +1,26 @@
 # Template file for 'ffms2'
 pkgname=ffms2
-version=2.23.1
-revision=3
-_version=2.23
-wrksrc="${pkgname}-${_version}"
+version=2.40
+revision=1
 build_style=gnu-configure
 configure_args="--enable-shared --disable-static"
-hostmakedepends="pkg-config"
+hostmakedepends="pkg-config autoconf automake libtool"
 makedepends="ffmpeg-devel zlib-devel"
-short_desc="A libav/ffmpeg based frames server"
+short_desc="Libav/ffmpeg based frames server"
 maintainer="DirectorX <void.directorx@protonmail.com>"
 license="MIT, GPL-3.0-or-later"
 homepage="https://github.com/FFMS/ffms2"
-distfiles="${homepage}/archive/${_version}.tar.gz"
-checksum=b09b2aa2b1c6f87f94a0a0dd8284b3c791cbe77f0f3df57af99ddebcd15273ed
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=82e95662946f3d6e1b529eadbd72bed196adfbc41368b2d50493efce6e716320
+
+case "$XBPS_TARGET_MACHINE" in
+	armv6l-musl) broken="https://github.com/void-linux/void-packages/runs/1584330257?check_suite_focus=true";;
+esac
```

Comment:
It isn't fine to stop building the package for a target, since it has dependants.

The issue here is the lack of `libatomic-devel`. It might be enough to add

```
if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
	makedepends+=" libatomic-devel"
fi
```

to the template.

Reply:
Otherwise, try adding `LDFLAGS="-latomic"` as well.